latex/Research_Strategy.tex

```diff
@@ -164,21 +164,22 @@
-Governance transitioned to an unfunded academic collaboration during SAMPL3 in 2012; this collaboration ran subsequent challenges as SAMPL4 (2014) and SAMPL5 (2016).
-The PI of this proposal (Mobley) was a primary organizer of SAMPL4--SAMPL5.
-SAMPL has always involved a component focused on calculation of solvation properties such as hydration free energies, but also introduced host-guest systems as model binding systems for SAMPL3-5, supplementing protein-ligand challenges (on trypsin and HIV integrase) which appeared in SAMPL3-4. 
-SAMPL has already been a tremendous resource for the community, resulting in roughly 100 publications (some are coming out as of this writing) which are typically cited 5-50 times or more each [full list of refs].
+\textbf{Physical modeling accuracy advances most rapidly when confronted by specific problems which must be modeled accurately, as revealed by carefully collected and curated data.}
+To make rapid progress, our field needs an effort which focuses on specific \emph{component} problems of the overall problem of interest, collects and curates data that highlights these problems, and drives progress via prospective challenges.
+This process allows the entire community to learn from both methodological success and failure. 
+The model we propose here has been proven to drive dramatic improvements in modeling, as evidenced by our {\bf Statistical Assessment of Modeling of Proteins and Ligands (SAMPL)} series of challenges. 
+SAMPL was born out of frustration with the lack of venues for comparing predictive accuracy on a level playing field, initiated by Anthony Nicholls of OpenEye software in 2007/2008~\cite{nicholls_predicting_2008}.
+Holding blind challenges and intensive workshops every two years since~\cite{nicholls_samp1_2009, mobley_predictions_2009, geballe_sampl2_2010, geballe_sampl3_2012, mobley_blind_2014-1, muddana_sampl4_2014, bannan_blind_2016, yin_sampl5_preprint}, governance transitioned to an unfunded academic collaboration during SAMPL3 (2012); this collaboration ran subsequent challenges as SAMPL4 (2014) and SAMPL5 (2016).
+The PI of this proposal (Mobley) was a primary organizer of SAMPL4/5 (2014--2016).
+SAMPL has always involved a component focused on calculation of solvation properties such as hydration free energies, but also introduced host-guest systems as model binding systems for SAMPL3--5 (2012--2016), supplementing protein-ligand challenges (on trypsin and HIV integrase) which appeared in SAMPL3-4 (2012--2014). 
+{\color{red}[JDC: This last sentence is weak, and doesn't reflect the early SAMPLs that had protein-ligand competitions where Marti Head individually bested computer docking programs. Can we replace this with a sentence that will pull its weight?]}
```

Comment:
Maybe, "SAMPL has historically focused on both simple physical property challenges such as hydration free energies, as well as host-guest binding and protein-ligand challenges on trypsin and integrase." Nothing on the other one was ever published (though it's possible to find something about it on two slides from an old CUP talk if you go desperately digging) so I think leaving it out is fair.